debian/postinst

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -eu
+
+g_s_helper_user=g-s-helper
+g_s_helper_home=/var/lib/${g_s_helper_user}
+
+# creating g-s-helper user if it isn't already there
+if ! getent passwd ${g_s_helper_user} >/dev/null; then
+    adduser --system --force-badname --quiet \
+            --no-create-home \
+            --home ${g_s_helper_home} \
+            --shell /usr/sbin/nologin \
+            --group ${g_s_helper_user}
+fi
+
+# create /var/lib/g-s-helper or update ownership of existing directory
+if [ -d ${g_s_helper_home} ]; then
+    chown -R ${g_s_helper_user}:${g_s_helper_user} ${g_s_helper_home}
+else
+    install -d -m0755 -o ${g_s_helper_user} -g ${g_s_helper_user} ${g_s_helper_home}
+fi
+
+# create external apps folders or update ownership of existing directory
+ext_apps_folder=/opt/flatpak-external-apps
```

Comment:
Why did you put this in `/opt`? I was thinking it could live in `/var/lib/flatpak-external-apps` next to the other canonical location.